src/main/java/net/imagej/ops/descriptor3d/DefaultCentroid.java

```diff
@@ -0,0 +1,46 @@
+package net.imagej.ops.descriptor3d;
+
+import org.scijava.plugin.Plugin;
+
+import net.imagej.ops.AbstractFunctionOp;
+import net.imagej.ops.Op;
+import net.imagej.ops.Ops.Descriptor3D;
+import net.imagej.ops.Ops.Descriptor3D.Centroid;
+import net.imglib2.Cursor;
+import net.imglib2.roi.IterableRegion;
+import net.imglib2.type.BooleanType;
+
+/**
+ * This {@link Op} computes the centroid of a {@link IterableRegion} (Label).
+ * 
+ * @author Tim-Oliver Buchholz, University of Konstanz.
+ *
+ * @param <B> a Boolean Type
+ */
+@Plugin(type = Op.class, name = Descriptor3D.Centroid.NAME)
```

Comment:
Do you need the `IterableRegion`? Can't you simply use the `CenterOfGravity` or `Centroid` from `Geometric2D'?
Maybe, we should introduce a namespace `geometric` comprising all n-dimensional implementations?
see also: https://github.com/imagej/imagej-ops/issues/222